sdk/core/core-amqp/src/connectionConfig/connectionConfig.ts

```diff
@@ -29,6 +29,14 @@ export interface ConnectionConfig {
    * @property {string} host - The host "<yournamespace>.servicebus.windows.net".
    */
   host: string;
+  /**
+   * @property {string} hostname - The hostname "<yournamespace>.servicebus.windows.net".
```

Comment:
Is the doc comment incorrect? What's the difference between 'host' and 'hostname'? Also, oof, these are _really_ similarly named. I would not be able to pick out the difference in context.

Reply:
I suppose another comment related - if they're the same why can't we just use 'host'?

Looking at how we use it later:

```typescript
hostname: parameters.config.hostname ?? parameters.config.host,
```

It seems like they should specify the same thing?

Reply:
Updated doc comments and the new field name (hostname to amqpHostname)